<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>689</width>
    <height>559</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ClipEdit</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="centralLayout">
    <item>
     <widget class="QFrame" name="main_menu">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>110</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>110</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-color:#d2d2d2;background-color:#E2E2E2;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <layout class="QVBoxLayout" name="menu_layout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>12</number>
       </property>
       <property name="topMargin">
        <number>8</number>
       </property>
       <property name="rightMargin">
        <number>12</number>
       </property>
       <property name="bottomMargin">
        <number>8</number>
       </property>
       <item>
        <widget class="QWidget" name="inner_main_menu" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="One_Menu_Button" name="button_arrow" native="true"/>
          </item>
          <item>
           <spacer name="menu_spacer01">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>8</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="One_Menu_Button" name="button_chart" native="true"/>
          </item>
          <item>
           <spacer name="menu_spacer02">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>8</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="One_Menu_Button" name="button_bullet" native="true"/>
          </item>
          <item>
           <spacer name="menu_spacer03">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>8</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="One_Menu_Button" name="button_clipart" native="true"/>
          </item>
          <item>
           <spacer name="menu_spacer04">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>8</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="One_Menu_Button" name="button_picture" native="true"/>
          </item>
          <item>
           <spacer name="menu_spacer05">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>8</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="One_Menu_Button" name="button_textbox" native="true"/>
          </item>
          <item>
           <spacer name="menu_spacer06">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>8</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="One_Menu_Button" name="button_screenshot" native="true"/>
          </item>
          <item>
           <spacer name="menu_spacer07">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>8</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="One_Menu_Button" name="button_layers" native="true"/>
          </item>
          <item>
           <spacer name="menu_spacer08">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>689</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menufile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="actionExports"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <widget class="QMenu" name="menuInsert">
     <property name="title">
      <string>Insert</string>
     </property>
     <addaction name="actionArrow"/>
     <addaction name="actionChart"/>
     <addaction name="actionClipart"/>
     <addaction name="actionNumberedBullets"/>
     <addaction name="actionPicture"/>
     <addaction name="actionTextBox"/>
    </widget>
    <addaction name="menuInsert"/>
    <addaction name="separator"/>
    <addaction name="actionScreenshot"/>
    <addaction name="separator"/>
    <addaction name="actionLayers"/>
   </widget>
   <addaction name="menufile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionExports">
   <property name="text">
    <string>Exports</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionArrow">
   <property name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/icons/icons/arrow-icon.png</normaloff>:/icons/icons/arrow-icon.png</iconset>
   </property>
   <property name="text">
    <string>Arrow</string>
   </property>
   <property name="toolTip">
    <string>Insert Arrow</string>
   </property>
  </action>
  <action name="actionChart">
   <property name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/icons/icons/chart-icon.png</normaloff>:/icons/icons/chart-icon.png</iconset>
   </property>
   <property name="text">
    <string>Chart</string>
   </property>
   <property name="toolTip">
    <string>Insert Chart</string>
   </property>
  </action>
  <action name="actionClipart">
   <property name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/icons/icons/clipart-icon.png</normaloff>:/icons/icons/clipart-icon.png</iconset>
   </property>
   <property name="text">
    <string>Cliparts</string>
   </property>
   <property name="toolTip">
    <string>Insert Clipart</string>
   </property>
  </action>
  <action name="actionNumberedBullets">
   <property name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/icons/icons/numbered-bullet-icon.png</normaloff>:/icons/icons/numbered-bullet-icon.png</iconset>
   </property>
   <property name="text">
    <string>Numbered Bullets</string>
   </property>
   <property name="toolTip">
    <string>Insert Numbered Bullets</string>
   </property>
  </action>
  <action name="actionPicture">
   <property name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/icons/icons/picture-icon.png</normaloff>:/icons/icons/picture-icon.png</iconset>
   </property>
   <property name="text">
    <string>Picture</string>
   </property>
   <property name="toolTip">
    <string>Insert Picture</string>
   </property>
  </action>
  <action name="actionTextBox">
   <property name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/icons/icons/textbox-icon.png</normaloff>:/icons/icons/textbox-icon.png</iconset>
   </property>
   <property name="text">
    <string>TextBox</string>
   </property>
   <property name="toolTip">
    <string>Insert TextBox</string>
   </property>
  </action>
  <action name="actionNewLayer">
   <property name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/icons/icons/layer-icon.png</normaloff>:/icons/icons/layer-icon.png</iconset>
   </property>
   <property name="text">
    <string>New layer</string>
   </property>
   <property name="toolTip">
    <string>Layers</string>
   </property>
  </action>
  <action name="actionRemoveLayer">
   <property name="text">
    <string>Remove layer</string>
   </property>
  </action>
  <action name="actionMergeDown">
   <property name="text">
    <string>Merge down</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as</string>
   </property>
  </action>
  <action name="actionScreenshot">
   <property name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/icons/icons/screenshot-icon.png</normaloff>:/icons/icons/screenshot-icon.png</iconset>
   </property>
   <property name="text">
    <string>Screenshot</string>
   </property>
   <property name="toolTip">
    <string>Take a screenshot</string>
   </property>
  </action>
  <action name="actionLayers">
   <property name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/icons/icons/layer-icon.png</normaloff>:/icons/icons/layer-icon.png</iconset>
   </property>
   <property name="text">
    <string>Layers</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>One_Menu_Button</class>
   <extends>QWidget</extends>
   <header>Components/one_menu_button.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
